module/userProjects/helper.js

```diff
@@ -1113,7 +1114,24 @@ module.exports = class UserProjectsHelper {
                         solutionDetails = solutionDetails.data[0];
                         
                     }
-
+                    
+                    // program join API call it will increment the noOfResourcesStarted counter and will make user join program
+                    // before creating any project this api has to called 
+                    let programJoinBody = {};
+                    programJoinBody.userRoleInformation = bodyData;
+                    programJoinBody.isResource = true;
+                    let joinProgramData = await coreService.joinProgram (
```

Comment:
@ankitpws As discussed, for the old app and old program core service API of the join program will always return success. We all understand that joining the program is a mandatory feature once this PII story is released. Showing a consent pop-up is optional, and all users after this feature is released will be marked as joined program if they start any resource in the current or upcoming program. I suggest we validate this understanding with the data and products team. Cc - @VISHNUDAS-tunerlabs